test/apiv2/20-containers.at

```diff
@@ -206,16 +206,6 @@ t POST containers/${cid_top}/stop "" 204
 t DELETE containers/$cid 204
 t DELETE containers/$cid_top 204
 
-# test the apiv2 create, shouldn't ignore the ENV and WORKDIR from the image
-t POST containers/create '"Image":"'$ENV_WORKDIR_IMG'","Env":["testKey1"]' 201 \
-  .Id~[0-9a-f]\\{64\\}
-cid=$(jq -r '.Id' <<<"$output")
-t GET containers/$cid/json 200 \
-  .Config.Env~.*REDIS_VERSION= \
-  .Config.Env~.*testKey1= \
-  .Config.WorkingDir="/data" # default is /data
-t DELETE containers/$cid 204
-
```

Comment:
cannot delete these lines

Reply:
@zhangguanzhang the problem is that when you submitted your initial PR, APIv2 tests were disabled in CI. They weren't running at all. When we tried to reenable those tests, the newly-added ones broke. I never reviewed that PR, and right now it is hard to guess the original intention, so for now the best approach is just to delete them and then add something back later that will work. I am really sorry for the hassle.

Reply:
ok, I will submit these after enabled APIv2 test